builder/osc/bsu/builder_acc_test.go

```diff
@@ -31,7 +31,7 @@ const testBuilderAccBasic = `
 		"type": "outscale-bsu",
 		"region": "eu-west-2",
 		"vm_type": "t2.micro",
-		"source_omi": "ami-6c274bbe",
+		"source_omi": "ami-6b9aeecd",
```

Comment:
Replace the OMI by one that will not be obsolete in the future.
On eu-west-2, we have duplicated the debian OMI: ami-68ed4301